package examples;

public class Main {

    public static void main(String[] args) {

        Exception exception = new Exception();
        exception.input();

	    /*int arr[] = {23, 6, 7};
	    //int x = 100 / 0;

        try {
            System.out.println(arr[3]);
            int x = 100 / 0;
        } catch (ArithmeticException e) {
            System.out.println("Error " + e);
        } catch (Exception e) {
            System.out.println("Error " + e);
        } finally {
            System.out.println("Finally is working");
        }*/
    }
}
